openpathsampling/sample.py

```diff
@@ -1,4 +1,4 @@
-import random
+from openpathsampling.random import random
```

Comment:
I take it this is the necessary way of getting to stdlib random? (Long term -- being before 2.0 -- everything in OPS should be using `paths.default_rng`)

Reply:
yes, just to make name-space clashes less likely